yarn/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Use the base App Engine Docker image, based on debian jessie.
+FROM gcr.io/google-appengine/debian8
+
+# Install updates and dependencies
+RUN apt-get update -y && apt-get install --no-install-recommends -y -q curl python build-essential git ca-certificates libkrb5-dev imagemagick && \
+    apt-get clean && rm /var/lib/apt/lists/*_*
+
+# Install the latest LTS release of nodejs
+RUN mkdir /nodejs && curl https://nodejs.org/dist/v6.9.1/node-v6.9.1-linux-x64.tar.gz | tar xvzf - -C /nodejs --strip-components=1
+ENV PATH $PATH:/nodejs/bin
+
+# Install the latest stable release of Yarn
+RUN mkdir /yarn && curl -L https://github.com/yarnpkg/yarn/releases/download/v0.24.6/yarn-v0.24.6.tar.gz | tar xvzf - -C /yarn --strip-components=1
+ENV PATH $PATH:/yarn/bin
```

Comment:
We have no preference there. It looks the same as what is done for the [npm builder](https://github.com/GoogleCloudPlatform/cloud-builders/blob/master/npm/Dockerfile#L10).